index.js

```diff
@@ -22,13 +22,19 @@ function copy(text, options) {
 
     mark = document.createElement('mark');
     mark.textContent = text;
-    // used to conserve newline, etc
-    mark.style.whiteSpace = 'pre';
-
-    // prevents scrolling to the end of the page
-    mark.style.position = 'fixed';
-    mark.style.top = '0';
-    mark.style.clip = 'rect(0px 0px 0px 0px)';
+    mark.style = [
+      // prevents scrolling to the end of the page
+      'position: fixed',
+      'top: 0',
+      'clip: rect(0, 0, 0, 0)',
+      // used to preserve spaces and line breaks
+      'white-space: pre',
+      // do not inherit user-select (it may be `none`)
+      '-webkit-user-select: text',
+      '-moz-user-select: text',
+      '-ms-user-select: text',
+      'user-select: text',
```

Comment:
Used `text` instead of `auto` because it's [definition](https://drafts.csswg.org/css-ui-4/#valdef-user-select-auto) is more straightforward